packages/flutter/test/widgets/focus_manager_test.dart

```diff
@@ -843,6 +844,17 @@ void main() {
           break;
       }
     }, variant: TargetPlatformVariant.all());
+    testWidgets('Mouse events change initial focus highlight mode.', (WidgetTester tester) async {
+      RendererBinding.instance.initMouseTracker();
```

Comment:
Is it worth expecting that these FocusHighlightModes start out as touch?

Reply:
Only on some platforms.  I'll add a check for that.